rdt/transformers/base.py

```diff
@@ -144,17 +144,31 @@ def _get_columns_data(data, columns):
         return data[columns].copy()
 
     @staticmethod
-    def _set_columns_data(data, columns_data, columns):
-        if columns_data is None:
-            return
+    def add_columns_to_data(data, columns, column_names):
```

Comment:
minor: can we keep it private (add the `_` back)? Sorry I didn't put that in my first comment

Reply:
:shipit: